### DISCUSSION TITLE
Support for the Internet Money wallet

### DISCUSSION THREAD

**Discussion by jonbulmer** [rainbow-me/rainbowkit#1460] [Feedback]
Hi I am using Rainbow Kit to build an app for the Internet Money wallet. It would be great if this wallet was a default option in the future. Thank you for the great experience.

**Comment by DanielSinclair** [comment]
@jonbulmer dApps can actually create their own wallet connector and use it as their first option for their dApp. Reference our [Custom Wallets](https://www.rainbowkit.com/docs/custom-wallets) guide.
